Dockerfile

```diff
@@ -0,0 +1,87 @@
+#
+# Dockerfile for an image with the currently checked out version of zipline installed. To build:
+#
+#    docker build -t quantopian/zipline .
+# 
+# To run the container:
+#
+#    docker run -v=/path/to/your/notebooks:/projects -p 8888:8888/tcp --name zipline -it quantopian/zipline
```

Comment:
I'm on Ubuntu 15.10, Chrome 49.0.2623.87.

I think using SSL by default makes sense, since it decreases the likelihood that someone runs this container publicly without securing it.  The flipside I suppose is that the person who's going to carelessly/unknowingly deploy this publicly probably isn't going to bother to change the password.  At the very least, it's nice to have the self-signed cert steps in code here, since they're a bit tricky.

One option might be to make it easier to run without a password via a flag like `--no-password`.  Doing that would probably require switching to using an ENTRYPOINT rather than a CMD though, which sounds like it breaks PyCharm in some way?  Another option might be to support setting a `NO_PASSWORD` envvar, and not pass `--certfile` and `--keyfile` if it's set.

My initial thought with all of the above though is that it's probably adding more complexity than its worth until someone actually runs into a problem trying to use this.